models/hop_protocol/optimism/hop_protocol_optimism_bridge_transfers.sql

```diff
@@ -0,0 +1,162 @@
+{{ config(
+    schema = 'hop_protocol_optimism',
+    alias = 'bridge_transfers',
```

Comment:
If we follow conventions from the other PR, this model would be named `hop_protocol_optimism_flows.sql`, I don't have strong opinions on this particular name (could also be `hop_protocol_optimism_bridge_flows.sql`) but we should standardize that

Reply:
yeah I'm good standardizing to `flows`